Prelude-UIKit/lenses/UIControlLenses.swift

```diff
@@ -78,27 +79,22 @@ public extension LensHolder where Object: UIControlProtocol {
 private func allTargetsSelectorsAndEvents(forControl control: UIControlProtocol)
   -> [TargetSelectorControlEvent] {
 
-    return control.allControlEvents().rawValue.bitComponents()
+    func bitComponents(forMask mask: UInt) -> [UInt] {
```

Comment:
no reason at all, just got fancy by moving it into that scope since it's only needed for that one function. moving back to less the noise in the diff. thanks!